src/components/backstage/plugins/Sidebar.js

```diff
@@ -231,39 +239,48 @@ const parseNpmData = (npmData) => {
     latestVersionPublishedAgo,
     firstPublishedAgo,
     lastSyncedAgo,
+    lastMonthDownloads,
+    numberOfVersions,
     maintainersHelpText,
   };
 };
 
-const Sidebar = ({ plugin }) => {
+const Sidebar = ({
+  plugin: {
+    availableOnRoadie,
+    roadieDocsPath,
+    npmPackageName,
+    codeLocation,
+  },
+}) => {
   const [npmData, setNpmData] = useState({});
   const [npmDataLoadingState, setNpmDataLoadingState] = useState('unloaded');
 
   useEffect(() => {
     (async () => {
       setNpmDataLoadingState('loading');
       const { status, data } = await fetchNpmDataByName({
-        packageName: plugin.npmPackageName,
+        packageName: npmPackageName,
       });
```

Comment:
There's no material changes here. I'm just exploding the `plugin` object up at the top of the function.